blocks/api/registration.js

```diff
@@ -44,6 +44,19 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( ! settings ||
+			( settings && typeof settings.save !== 'function' ) ) {
```

Comment:
Could we write this like?

```js
if ( 
  ! settings  ||
  ( settings && typeof settings.save !== 'function' )
)
```

I think this pattern is the one used in the Gutenberg code